graphql/admin/admin.go

```diff
@@ -502,9 +502,10 @@ var (
 	adminQueryMWConfig = map[string]resolve.QueryMiddlewares{
 		"health":       minimalAdminQryMWs, // dgraph checks Guardian auth for health
 		"state":        minimalAdminQryMWs, // dgraph checks Guardian auth for state
-		"config":       stdAdminQryMWs,
+		"config":       gogQryMWs,
 		"listBackups":  gogQryMWs,
 		"getGQLSchema": stdAdminQryMWs,
+		//	"getLambdaScript": stdAdminQryMWs,
```

Comment:
this is unrelated to the PR being cherry-picked (see discussion in PR).  Adding comment to preserve this discussion for context

Reply:
> I see, yep they have changed a lot of places #7955 Also need to continue the change in the lambda repo.
> 
> In 21.03 (and previous versions) we stored the lambda scripts directly via flag in the binary. This change to GraphQL makes easy to deal with the JSscript(it is stored in bas64). This seemed obvious to me. It's a change from 16 months ago. I assumed it was already in effect.
> 
> As you can see, it is present in v21.09-slash(a cloud version) https://github.com/dgraph-io/dgraph/blob/release/v21.09-slash/graphql/admin/admin.go

That PR link is useful, thanks for the context @MichelDiz .  I am removing this line since this change is unrelated to the lambda changes in the other PR.